src/files/pyfingerprint/pyfingerprint.py

```diff
@@ -997,20 +997,18 @@ def loadTemplate(self, positionNumber, charBufferNumber = 0x01):
         else:
             raise Exception('Unknown error')
 
-    def deleteTemplate(self, positionNumber):
+    def deleteTemplate(self, positionNumber, count = 1):
```

Comment:
It is always better to validate it to avoid possible malfunction of the sensor hardware.